RELEASING.md

```diff
@@ -1,88 +1,259 @@
-
-## Preparing for a release
-1. Ensure master docs are passing semaphore tests.
-1. Check out latest master locally.
-
-## Creating the release
-### When doing a major/minor release ONLY
-These steps are for creating a new major/minor release, with the expectation
-that a release candidate will be created, published and tested before the final
-release is announced.
-
-1. Run `./release-scripts/do_release.py` to copy master directory to the new version directory
-1. Add the new version to `_data/versions.yml` (add the date of the release in the description)
-   - Add new versions that are a release candidate to the end of the file.
-     If a release is added to the top of the file it will become the 'Latest Release'.
-   - Make sure new release candidate versions are added to the top of the new release section (otherwise
-     yaml will incorrectly identify an older release candidate):
-     ```
-      v2.X
-        - title: v2.X.0-rc2
-          ... etc ...
-
-        - title: v2.X.0-rc1
-          ... etc ...
-     ```
+# How to release Calico
+
+> **NOTE:** These instructions apply only to Calico versions v3.2.0 or greater.
+> For older releases, refer to the instructions in the corresponding `release-vX.Y` branch.
+
+## Overview
+
+This repository contains documentation and packaging, but no Calico code.
+
+Releases of this repository still serve several important purposes. Namely, they provide:
+
+- a single Calico release with user-facing release notes and documentation.
+- a packaging of individual Calico component releases into a single `.tgz` file
+
+## Prerequisites
+
+Before attempting to create a Calico release you must do the following.
+
+1. Choose a Calico version number, e.g. `v3.2.0`.
+
+1. Select the appropriate component version numbers, and create any necessary releases.
+
+   The following components _must_ use the same version number as the Calico version number
+   chosen above, and thus must be re-released for every Calico release.
+
+   - [calico/node](https://github.com/projectcalico/node/releases)
+   - [calico/ctl](https://github.com/projectcalico/calicoctl/releases)
+   - [calico/cni](https://github.com/projectcalico/cni-plugin/releases)
+   - [calico/kube-controllers](https://github.com/projectcalico/kube-controllers/releases)
+   - [calico/felix](https://github.com/projectcalico/felix/releases)
+
+   The following components do not yet use the same versioning as the Calico repo, and
+   so the version to include may differ.
+
+   - [calico/typha](https://github.com/projectcalico/typha/releases)
+   - [networking-calico](https://github.com/projectcalico/networking-calico)
+
+   The following components do not share a version with the Calico release, but are included in the documentation.
+
+   - [flannel](https://github.com/coreos/flannel/releases)
+
+## Preparing to cut a Calico release
+
+
+1. Check out the `master` branch of this repository and make sure it is up-to-date and passing Semaphore CI.
```

Comment:
a link to semaphore's CI for calico master would be nifty: https://semaphoreci.com/calico/calico/branches/master